test/cli/repo.js

```diff
@@ -18,4 +19,30 @@ describe('repo', () => runOnAndOff((thing) => {
       expect(out).to.eql(`${repoVersion}\n`)
     })
   })
+
+  // Note: There are more comprehensive GC tests in interface-js-ipfs-core
+  it('should run garbage collection', async function () {
```

Comment:
This test should be moved to core or interface-core.
Here we should test these lines https://github.com/ipfs/js-ipfs/blob/8c21da09615dcc12ad91bc7eec96ca5567a37de7/src/cli/commands/repo/gc.js#L26-L32 using the same approach used here https://github.com/ipfs/js-ipfs/blob/master/test/cli/name.js

We will make some changes during this quarter to make tests like these even simpler, faster and cleaner (reducing mocking and those pesky `clearModule`).

Reply:
Thanks that's very helpful - I added CLI tests based on this example